elisp/intero.el

```diff
@@ -904,6 +921,8 @@ If specified, MINLEN is the shortest completion which will be
 considered."
   (when (intero-completions-can-grab-prefix)
     (let ((prefix (cond
+                   ((and (eq 'intero-repl-mode major-mode)
```

Comment:
Prefer `(derived-mode-p 'intero-repl-mode)` to testing for exact equality of `major-mode`.

Reply:
Generally you'd put the test for applicability (i.e. the major mode test here) in the function which grabs the prefix: it would return nil if inapplicable.